photon-lib/src/test/scala/com/linkedin/photon/ml/optimization/LBFGSTest.scala

```diff
@@ -40,23 +40,18 @@ class LBFGSTest {
     Array(
       // Note that expected value here is the value of the function at the unconstrained optima since the
       // projection happens after it
-      Array(Some(Map[Int, (Double, Double)]()), Array(4.0)),
-      Array(Some(Map[Int, (Double, Double)](0 -> (3.0, 5.0))), Array(4.0)),
-      Array(Some(Map[Int, (Double, Double)](0 -> (Double.NegativeInfinity, 3.0))), Array(3.0)),
-      Array(Some(Map[Int, (Double, Double)](0 -> (5.0, Double.PositiveInfinity))), Array(5.0))
+      Array(Array(4.0))
     )
   }
 
   @Test(dataProvider = "dataProvider")
```

Comment:
Since there's only one test now, we may as well get rid of the data provider and just check that it optimizes towards `TestObjective.CENTROID`.